client/my-sites/stats/all-time-highlights-section/post-cards-group.tsx

```diff
@@ -53,40 +54,41 @@ const getStatsQueries = createSelector(
 );
 
 const getStatsData = createSelector(
-	( state, siteId, topPostsQuery ) => {
+	// eslint-disable-next-line @typescript-eslint/no-unused-vars
+	( state, siteId, topPostsQuery, isTopViewedPostRequesting ) => {
 		const { post: topPost, page: topPage } = getTopPostAndPage( state, siteId, topPostsQuery );
 
 		return {
 			topPost,
 			topPage,
 		};
 	},
-	( state, siteId, topPostsQuery ) => [ topPostsQuery ]
+	( state, siteId, topPostsQuery, isTopViewedPostRequesting ) => [
+		topPostsQuery,
+		isTopViewedPostRequesting,
+	]
```

Comment:
It might be nice to move the "Refresh memoized data from `createSelector` with the loading flag" code comment to annotate this block.